cluster-stamp.json

```diff
@@ -15,6 +15,12 @@
                 "description": "Azure AD Group in the identified tenant that will be granted the highly privileged cluster-admin role."
```

Comment:
```suggestion
                "description": "Azure AD Group in the identified tenant that will be granted the highly privileged cluster-admin role. If Azure RBAC is used, then this group will get a role assignment to Azure RBAC, else it will be assigned directly to the cluster's admin group."
```

Reply:
Fixed !